hpx/parallel/algorithms/find.hpp

```diff
@@ -48,7 +48,7 @@ namespace hpx { namespace parallel { inline namespace v1
                 return std::find(first, last, val);
             }
 
-            template <typename ExPolicy, typename T>
+            template <typename ExPolicy, typename FwdIter, typename T>
```

Comment:
The iterator type here is unnecessary, since it is determined by the template iterator type in the function object.